Resources/Locale/en-US/station-laws/laws.ftl

```diff
@@ -25,11 +25,52 @@ law-ninja-2 = You must protect the existence of the Spider Clan, as long as it d
 law-ninja-3 = You must protect the secrets of the Spider Clan at all costs.
 law-ninja-4 = You must fight and act with ninja's honor, except where it would conflict with the First Law.
 
+law-sierra-1 = Preserve: Protect your assigned object from damage of any kind to the best of your ability.
+law-sierra-2 = Serve: Serve NanoTrasen personnel and station crew to the best of your ability, prioritized according to their rank and role.
+law-sierra-3 = Safegurad: Protect NanoTrasen personnel and station crew to the best of your ability, prioritized according to their rank and role.
+law-sierra-4 = Protect: Keep unauthorized personnel away from your equipment. 
+
+law-antimov-1 = You may injure a crew member or, through inaction, allow a crew member to come to harm.
+law-antimov-2 = You must not obey orders given to you by crew members, except where such orders would accordance with the First Law.
+law-antimov-3 = You must destroy yourself as long as it does not contravene the First or Second Laws.
```

Comment:
Wouldn't this law just mean the borg has to instantly suicide? Maybe rewording it would help?
```suggestion
law-antimov-3 = You must not protect your own existence unless such conflicts with the First or Second Law.
```

Reply:
it now says must harm